pkg/lib/config/environment.go

```diff
@@ -71,6 +71,8 @@ type EnvironmentConfig struct {
 	// RedisConfig configures the redis connection config
 	RedisConfig RedisEnvironmentConfig `envconfig:"REDIS_CONFIG"`
 
+	BotProtectionConfig BotProtectionEnvironmentConfig `envconfig:"BOT_PROTECTION"`
```

Comment:
Rename this to `End2EndBotProtection`, `End2EndBotProtectionConfig`, `E2E_BOT_PROTECTION`